apps/frontend/src/app/3_organisms/ZeroLocForm/components/OpenCreditLine.tsx

```diff
@@ -44,6 +44,7 @@ export const OpenCreditLine: FC<OpenCreditLineProps> = ({
     useAmountInput('');
   const [debtAmountInput, setDebtAmount, debtAmount] = useAmountInput('');
   const [debtToken, setDebtToken] = useState<SupportedTokens>(BORROW_ASSETS[0]);
+  const [maxOriginationFeeRate, setMaxOriginationFeeRate] = useState('0');
```

Comment:
The same problem as with `AdjustCreditLine` - use `maxBorrowingFeeRate` as the initial value.